src/pages/workspaces/WorkspacesList/NoWorkspacesMessage.ts

```diff
@@ -0,0 +1,34 @@
+import { FC, Fragment } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { NoWorkspacesMessage } from 'src/components/workspace-utils';
+import { cond } from 'src/libs/utils';
+
+import { CatagorizedWorkspaces } from './CatagorizedWorkspaces';
+import { WorkspaceFilterValues } from './WorkspaceFilters';
+import { updateWorkspaceActions } from './WorkspaceUserActions';
+
+interface NoContentMessageProps {
+  loadingWorkspaces: boolean;
+  loadingSubmissionStats: boolean;
+  workspaces: CatagorizedWorkspaces;
+  filters: WorkspaceFilterValues;
+}
+
+export const NoContentMessage: FC<NoContentMessageProps> = ({
```

Comment:
@nawatts can you remind me of what we have settled on for component properties? I personally prefer something like this (don't like repeating the individual properties and like referencing them through props to make it clear where they came from), but I don't remember where we landed on this in the UIWG.

```
interface WorkspaceAttributeNoticeProperties {
  accessLevel: WorkspaceAccessLevel;
  isLocked: boolean;
  workspaceProtectedMessage?: string;
  workspaceRegionConstraintMessage?: string;
}

const WorkspaceAttributeNotice = (props: WorkspaceAttributeNoticeProperties) => {

Reply:
I believe the current recommendation is:

```ts
interface ComponentProps {
  ...
}

export const Component = (props: ComponentProps): ReactNode => {
  ...
}
```

Generally,
- Prefer `interface` over `type` for props for better error messages from TS
- Add `ReactNode` as return type instead of using `FC` more more consistent type signatures across all components (`FC` won't work if the props type is parameterized)
- Avoid destructuring in the function arguments to make the type signature easier to read. If you want to destructure `props`, do that as the first line in the function body.

Reply:
yup, great rundown Nick.  Let me check the guide docs to make sure all this got in.